openmdao/utils/units.py

```diff
@@ -845,6 +845,11 @@ def _update_library(cfg):
 _UNIT_CACHE = {}
 
 
+def _is_unitless(units):
```

Comment:
I don't think this gives the right answer.  For me it returns True for 'ft/s'.   Also, PhysicalUnit already has an is_dimensionless() method, so you should just be able to use that,  `return unit_meta.is_dimensionless()`

Reply:
Fixed